apps/jobs/migrations/0025_remove_submission_static_code_upload_environment_log_file.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.20 on 2022-11-07 06:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('jobs', '0024_submission_static_code_upload_submission_log_file'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='submission',
+            name='static_code_upload_environment_log_file',
+        ),
+    ]
```

Comment:
Something seems incorrect with these migration files. Can you remove the new migration files you generated in this PR once and regenerate them? There should ideally be just 1 new migration file

Reply:
Oh, I see what you mean. I'll rebuild those migrations specifically.

Reply:
Done